events/templates/event.html

```diff
@@ -42,17 +43,44 @@ <h6 class="float-right inline"> posted by:
     <!-- Users Signed Up -->
     {% with signed_up=event.signed_up.all %}
       <p>Number signed up: {{ signed_up.count }}</p>
-      <table class="table table-hover">
-        {% for atendee in signed_up %}
-          <tr>
-            <td>
-              <a href="{% url 'profile' atendee.id %}">
-                {{atendee.username}}
-              </a>
-            </td>
-          </tr>
-        {% endfor %}
-      </table>
+
+        <!-- Button trigger modal -->
+        <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#exampleModalLong">
+          Users Attending
+        </button>
+
+        <!-- Modal -->
+        <div class="overflow-auto">
+          <div class="modal fade" id="exampleModalLong" tabindex="-1" role="dialog" aria-labelledby="exampleModalLongTitle" aria-hidden="true">
```

Comment:
There is an issue with this naming scheme. The IDs of all modals are all the same so when theres a list of many modals (like on the home page when showing multiple events), every button is triggering the same modal (the first one I think).
Im not certain about a fix for this. Perhaps try generating a unique ID using the {{ event.id }}